guide/additional-info/changes-in-v14.md

```diff
@@ -0,0 +1,132 @@
+# Updating from v13 to v14
+
+## Before you start
+
+v14 requires Node 16.9 or higher to use, so make sure you're up to date. To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it! There are many resources online to help you with this step based on your host system.
+
+## Breaking Changes
+
+### Enum Values
+
+Any areas that used to accept a `string` or `number` type for an enum parameter will now only accept exclusively `number`s.
+
+In addition, the old enums exported by discord.js v13 and lower are replaced with new enums from discord-api-types (link here).
+
+<details>
+<summary> New enum differences </summary>
+  Most of the difference between enums from discord.js and discord-api-types can be summarized as so:
+
+1. Enums are singular ie `ApplicationCommandOptionTypes` -> `ApplicationCommandOptionType`
+2. Enums that are prefixed with `Message` no longer have the `Message` prefix ie `MessageButtonStyles` -> `ButtonStyle`
+3. Enum values are `PascalCase` rather than `SCREAMING_SNAKE_CASE` ie `.CHAT_INPUT` -> `.ChatInput`
+ </details>
+
+There are two recommended ways of representing enum values:
+
+1. Use the actual enum type: `ButtonStyle.Primary`
+2. Continue using a string representation but instead use the new `EnumResolver`:
+
+```js
+const { EnumResolver } = require('discord.js');
+const enumValue = EnumResolvers.resolveButtonStyle('PRIMARY');
+```
+
+Areas like JSON slash commands and JSON message compononents will likely need to be modified to accomodate these changes:
+
+**Application commands**:
+
+```diff
++ const { ApplicationCommandType, ApplicationCommandOptionType } = require('discord.js');
+
+const command = {
+  name: 'ping',
+- type: 'CHAT_INPUT',
++ type: ApplicationCommandType.ChatInput
+  options: [
+    name: 'option',
+    description: 'A sample option'
+-   type: 'STRING',
++   type: ApplicationCommandOptionType.String
+  ],
+};
+```
+
+**Buttons**:
+
+```diff
++ const { ButtonStyle } = require('discord.js');
+
+const button = {
+  label: 'test'
+- style: 'PRIMARY'
++ style: ButtonStyle.Primary
+  customId: '1234'
+}
+```
+
+### Event Removals
+
+`message` and `interaction` events are now removed. Use `messageCreate` and `interactionCreate` instead.
+
+`applicationCommandCreate`, `applicationCommandDelete` and `applicationCommandUpdate` have all been removed. Refer to [this PR](https://github.com/discordjs/discord.js/pull/6492) for context.
+
+# Utility Changes
+
+`Util#removeMentions` has been removed.
+
+### CDN Changes
+
+Methods that return CDN URLs will now return a dynamic image URL (if available). This behavior can be overriden by setting `forceStatic` to `false` in the `MakeURLOptions` parameters.
+
+### Guild Object Changes
+
+All `Guild#setXPositions` methods have been removed, use `RoleManager#setPositions` and `GuildChannelManager#setPositions` instead.
```

Comment:
Instead of saying:

> All `Guild#setXPositions` methods have been removed

I would actually list out what was removed. It feels strange you haven't since you've done it later for the replacements.

```suggestion
`Guild#setRolePositions` and `Guild#setChannelPositions` have been removed. Use `RoleManager#setPositions` and `GuildChannelManager#setPositions` instead respectively.
```